test/integration/node-specific/errors.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from 'chai';
+
+import { MongoClient, MongoError } from '../../mongodb';
+
+describe.only('Error (Integration)', function () {
```

Comment:
```suggestion
describe('Error (Integration)', function () {
```

![GO_TO_JAIL](https://github.com/mongodb/node-mongodb-native/assets/10873993/69b4a8a1-1fbc-4b15-b0d9-eea964a84a46)